### ISSUE TITLE
[BUG] Website documentation outdated?

### ISSUE THREAD

**Issue by roman-petrov** [dart-code-checker/dart-code-metrics#521]
In this repository README there are command line options like:

```
--[no-]fatal-style       Treat style level issues as fatal.
--[no-]fatal-performance Treat performance level issues as fatal.
--[no-]fatal-warnings    Treat warning level issues as fatal.
```

These options are also reported in help text when running `metrics analyze`. I not see these options on [website](https://dartcodemetrics.dev/docs/cli/analyze), am I missing something or website is a bit outdated?
